jest.setup.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs');
 const { join } = require('path');
 const { getAccountDb } = require('./account-db');
-const config = require('./load-config');
+const { default: config } = require('./load-config');
```

Comment:
Is it possible to switch this file to ESM too? real ESM files can’t be `require()`-ed synchronously.